ui/src/app/dashboard/views/Dashboard.tsx

```diff
@@ -1,24 +1,31 @@
 import { Route, Switch } from "react-router-dom";
 
 import DashboardConfigurationForm from "./DashboardConfigurationForm";
+import DashboardHeader from "./DashboardHeader";
 import DiscoveriesList from "./DiscoveriesList";
 
+import Section from "app/base/components/Section";
 import NotFound from "app/base/views/NotFound";
 import dashboardURLs from "app/dashboard/urls";
 
 const Dashboard = (): JSX.Element => {
   return (
-    <Switch>
-      <Route exact path={[dashboardURLs.index]}>
-        <DiscoveriesList />
-      </Route>
-      <Route exact path={[dashboardURLs.configuration]}>
-        <DashboardConfigurationForm />
-      </Route>
-      <Route path="*">
-        <NotFound />
-      </Route>
-    </Switch>
+    <Section
+      header={<DashboardHeader />}
+      headerClassName="u-no-padding--bottom"
+    >
+      <Switch>
+        <Route exact path={[dashboardURLs.index]}>
+          <DiscoveriesList />
+        </Route>
+        <Route exact path={[dashboardURLs.configuration]}>
+          <DashboardConfigurationForm />
+        </Route>
+        <Route path="*">
+          <NotFound />
+        </Route>
+      </Switch>
+    </Section>
```

Comment:
@petesfrench @steverydz I've updated this component so that the Section and DashboardHeader components are consistent between routes, and only the content below the header changes with the route.